apis/Google.Cloud.AspNetCore.DataProtection.SecretManager/Google.Cloud.AspNetCore.DataProtection.SecretManager/Google.Cloud.AspNetCore.DataProtection.SecretManager.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <Version>1.0.0-alpha01</Version>
+        <TargetFramework>netstandard2.1</TargetFramework>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <Description>Recommended Google client library to use Google Cloud Secret Manager to store data protection keys for ASP.NET Core.</Description>
+        <PackageTags>Secret Manager;Google;Cloud;Security</PackageTags>
+    </PropertyGroup>
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
Do we need this section? Please remove them.

Reply:
done